### PR TITLE
Add privacy ding and band to product page intro section

### LINKED ISSUE #5238: Add "privacy" ding to upper portion of product page
### Todo
- [x] Add large privacy ding to hero when applicable - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [x] Add ‘Warning: *privacy not included with this product’ bar when applicable - #FBD545 / use font styles already in on production
- [x] Add mini privacy ding to `What could happen if something goes wrong?` - #5250
- [ ] ~~Add mini privacy ding when applicable (i.e. when the answer is `Yes` to the ding questions related to "How does it use this data?","How can you control your data?","What is the company’s known track record of protecting users’ data?","Does this product meet our Minimum Security Standards?") - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)~~ Move to https://github.com/mozilla/foundation.mozilla.org/issues/5308

### Top section desktop
image

### Top section mobile
Screen Shot 2020-09-18 at 1 00 15 PM

### Criteria section desktop
image

### Criteria section mobile
image

---

**🗒  Design -> Dev Handoff Checklist**

- [x] Design has been finalized
- [x] Link to original design ticket #5152
- [X] Link to related ticket #5234
- [x] Link to [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included - [icon](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [x] Hover state for elements are designed - n/a
- [x] Other states for elements are designed (e.g., error state for form) - n/a

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5238

Add ding to hero and a notice band applicable.

- no ding: https://foundation-s-issue-5238-8affgg.herokuapp.com/en/privacynotincluded/products/teacher-scene/
- has ding: https://foundation-s-issue-5238-8affgg.herokuapp.com/en/privacynotincluded/products/rather-beyond/

### CODE REVIEW

File: source/sass/buyers-guide/views/product.scss [hunk 1]
```diff
@@ -11,6 +11,27 @@ $pni-product-breakpoint-larger: $bp-md;
...
+
+  .product-header-content.show-privacy-ding {
+    position: relative;
+
+    &:after {
+      content: "";
+      position: absolute;
+      top: 24px;
+      right: 24px;
+      background: url(../_images/buyers-guide/icon-privacy-ding.svg) center /
```
Comment (mmmavis) [inline source/sass/buyers-guide/views/product.scss:23]: We have a band that reads `Warning: *privacy not included with this product` already so I'm treating this badge as purely decorative image and therefore I implemented it as a pseudo element.

Comment (Pomax) [reply source/sass/buyers-guide/views/product.scss:23]: yep, makes sense to me

File: source/sass/mofo-bootstrap/_colors.scss [hunk 2]
```diff
@@ -27,6 +27,7 @@ $pni-purple: #7f28b7;
 $pni-purple-pink: #b0379b;
 $pni-pink: #e4487d;
 $pni-gradient: linear-gradient(to right, $pni-blue 0%, $pni-pink 100%);
+$pni-warning-yellow: #fbd545;
```
Comment (Pomax) [inline source/sass/mofo-bootstrap/_colors.scss:30]: would it be worth making a `_colors__pni.scss` and importing that?

**Review by Pomax — APPROVED** [review]
R+, one question around scss organization but nothing that would prevent this from landing.

File: source/sass/mofo-bootstrap/_colors.scss (hunk 2 above)
Comment (mmmavis) [reply source/sass/mofo-bootstrap/_colors.scss:30]: Yup can do that. But will file a new ticket for it.

Comment (mmmavis) [reply source/sass/mofo-bootstrap/_colors.scss:30]: Fileld https://github.com/mozilla/foundation.mozilla.org/issues/5315

**Comment by kristinashu** [comment]
This is indeed very exciting to see!

For "Warning: *privacy not included with this product" text, it looks like the PR is using 18/27 (the size is the bigger body of the intro blurb/what could happen) but the mockup uses the slightly small text 16/24 (the size of the body of the main criteria copy.) Please update to the smaller text size. Otherwise, everything looks great!

**Review by kristinashu — CHANGES_REQUESTED** [review]
Font size update.

**Comment by mmmavis** [comment]
PR updated!

**Review by kristinashu — APPROVED** [review]
Nice!

File: source/sass/buyers-guide/views/product.scss (hunk 1 above)
Comment (TheoChevalier) [reply source/sass/buyers-guide/views/product.scss:23]: Note that localized versions of this image are coming (see https://github.com/mozilla/foundation.mozilla.org/issues/5260) and that it’s probably easier to handle if the image is set in the template.
We can still play with the attribute selector `html[lang="fr"]`, tough

**Pull request merged**